src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -343,6 +578,32 @@ public int RunTest(string executable, string outputFile, string errorFile, strin
                         exitCode = process.ExitCode;
                         MobileAppHandler.CheckExitCode(exitCode, testBinaryBase, category, outputWriter);
                         Task.WaitAll(copyOutput, copyError);
+
+                        if (!OperatingSystem.IsWindows())
+                        {
+                            // crashreport is only for non-windows.
+                            if (exitCode != 0)
+                            {
+                                // Search for dump, if created.
+                                if (Directory.Exists(crashDumpFolder))
+                                {
+                                    outputWriter.WriteLine($"Test failed. Trying to see if dump file was created in {crashDumpFolder} since {startTime}");
```

Comment:
We should think of a way to do this smarter - timestamps don't work great with multiple tests running at the same time.

Reply:
I couldn't think of a better way. Let me know if we can do something so that we can get back the dump name or something.